proto/storage/compliance_operator_v2.proto

```diff
@@ -135,6 +135,26 @@ message ComplianceOperatorClusterScanConfigStatus {
   string cluster_name = 5;
 }
 
+// Next Tag: 7
+message ComplianceOperatorBenchmarkV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Benchmark Name,hidden"
+  string version = 3;
+  string description = 4;
+  string provider = 5;
+  // Example: https://control.compliance.openshift.io/CIS-OCP
+  string profile_annotation = 6; // @gotags: search:"Compliance Profile Annotation";
+  // This is extracted from the annotation. Example: CIS-OCP
+  string short_name = 7; // @gotags: search:"Compliance Benchmark Short Name,hidden"
```

Comment:
IMO, I think we only. need one of these.  Probably the `short_name`.  The first part of the annotation prefix could simply be a constant as it is elsewhere that we pull annotation values.

Reply:
It make sense. I think the full annotation is stored in the rules and we can also have a constant in the code somewhere with the first part (`control.compliance.openshift.io/`). I'll go ahead and remove the `profile_annotation` and keep only the `short_name`.